ratpack-core/src/main/java/ratpack/http/internal/DefaultStatus.java

```diff
@@ -41,4 +41,9 @@ public String getMessage() {
   public HttpResponseStatus getNettyStatus() {
     return status;
   }
+
+  @Override
+  public String toString() {
+    return status.code() + ":" + status.reasonPhrase();
```

Comment:
Looking at `io.netty.handler.codec.http.HttpResponseStatus` reason phrase cannot be null it will bomb out in the constructor. If you use `HttpResponseStatus(int code)` one is created from the default phrase for that code + the code.